internal/k8s/ako_init.go

```diff
@@ -1226,6 +1226,17 @@ func (c *AviController) FullSyncK8s(sync bool) error {
 			}
 		}
 
+		// Proxy Enabled Application Profile GET/CREATE/UPDATE
+		aviClientPool := avicache.SharedAVIClients(lib.GetAdminTenant())
+		if aviClientPool == nil || len(aviClientPool.AviClient) == 0 {
+			return fmt.Errorf("avi Rest client initialization failed")
+		}
+		err = lib.ProxyEnabledAppProfileCU(aviClientPool.AviClient[0])
```

Comment:
@Rishabh-Baranwal @abhishekbsingh Should we include this AppProfile creation flow as part of infra container? We already creating SEG, network in that. So resource creation for wcp will remain in one container only.

Reply:
We need this in case of VDS as well.